### ISSUE TITLE
Align json for data transfer

### ISSUE THREAD

**Issue by gidili** [MetaCell/geppetto-scidash#30]
Hi @rgerkin as discussed [in this card](https://github.com/scidash/sciunit/issues/64), here's the current version of the json we are using. I am posting it here as it might be easier for you to work with this as the target:

https://github.com/MetaCell/scidash-api/blob/master/json_sample.json

Summary of changes thus far:

- sort_key numerical attribute to be added to score (this is normalised version of the actual score that we need to color map how good a score is from 0 to 1)
- score_type (_class) string attribute to be added to score (shows up in json as '_class': 'ZScore' for the score object)
- test class to be changed by Rick to an object (at the moment it's '_class': 'MyTest' on the test instance) to have a name and an optional url
- model class also same as above for test class
- observation can have an optional url attribute

**Comment by rgerkin** [comment]
The property is called `sort_key` not `score_key`.  At least, this is what it is called in SciUnit.  

As for the others, are you saying you want sciunit (in python) to export JSON with different attribute names?  Why can't the code on your side just use the names as they are currently?

**Comment by gidili** [comment]
@rgerkin good catch on sort_key, thanks for reviewing. 

About the other question, don't worry about changing the others attribute names, I meant to convey that we'll do the mapping in scidash-api, it's good to have one place where we can cope with different names if any of them change in the future anyway. 

The names in our json are different because they follow the domain entities naming we used to fully qualify them rather than just use whatever the serialisation spits out with underscores etc.

So basically as long as the structure/hierarchy of the json is the same that's all we need, we can deal with the attribute discrepancies.

**Comment by rgerkin** [comment]
I think it seems like the right structure/hierarchy.

**Comment by gidili** [comment]
Hi @rgerkin just checking if the latest on serialization is still this example? --> https://github.com/scidash/neuronunit/blob/dev/neuronunit/unit_test/serialization_test.ipynb

Doesn't seem to have the changes outlined above for test/model class (object instead of string) but maybe you haven't gotten around to doing that yet or I am looking in the wrong place.

**Comment by gidili** [comment]
Not a blocker for us btw, just checking-in, no pressure 👍

**Comment by rgerkin** [comment]
Working on this now.

**Comment by rgerkin** [comment]
Implemented so far, locally (I will push when all of these are done). 
- [x] sort_key in json
- [ ] score_type in json
- [ ] test class as an object with depth (not a string)
- [ ] model class as an object with depth (not a string)
- [x] observation can have an optional url attribute

**Comment by rgerkin** [comment]
OK, I believe all these things are fixed.  Check the file now.  I kept the word '_class' in for unrelated reasons, but all the other things you need are there.

**Comment by gidili** [comment]
Thanks @rgerkin! Looks good 👍 Will let you know if any questions, closing this for now.
